php/blocks/class-field.php

```diff
@@ -194,6 +194,17 @@ public function cast_value( $value ) {
 	 * @return string $value The value to echo.
 	 */
 	public function cast_value_to_string( $value ) {
+		/**
+		 * Filters the value of a field when being output as a string.
+		 *
+		 * The result of this filter should be a string. Even if it isn't, it will be run
+		 * through strval() anyway.
+		 *
+		 * @param mixed $value  The value to cast as a string.
+		 * @param Field $field  This field.
+		 */
+		$value = apply_filters( 'block_lab_cast_field_value_to_string', $value, $this );
```

Comment:
This filter looks good, great documentation of it.

What do you think of moving it to right above the return statement:

```php
return strval( $value );
```

Reply:
Actually, don't worry about this comment for now.